packages/notebook/src/actions.tsx

```diff
@@ -549,7 +549,10 @@ export namespace NotebookActions {
       translator
     );
 
-    void Private.handleRunState(notebook, state, false);
+    // While we do not want to scroll if cell is visible,
+    // we still need to invoke the scrolling logic
+    // in case if cell is outside of viewport.
```

Comment:
How does one execute the run command on a cell outside of the viewport?

Reply:
Many possibilities, one is the ToC:

![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/3d6236b2-e172-405d-a9ee-5f2f785c0e2b)

Reply:
I think that we should also have a function that does not do mode changing nor scrolling (and some extension developers have specifically asked about it), but that might not even use `handleRunState` at all and is out of scope for this PR. I removed the `scroll` argument in 39ce21d1927e71b37dad7be1891b2fbdb103634f.

Reply:
By "function that does not do mode changing nor scrolling", do you mean, for example, being able to run a cell but stay in edit mode and do not scroll?